software/tubearchivist.yml

```
@@ -0,0 +1,16 @@
+name: "Tube Archivist"
+website_url: "https://tubearchivist.com/"
+source_code_url: "https://github.com/tubearchivist/tubearchivist"
+description: "Organize, search, and enjoy your YouTube collection. Subscribe, download, and track viewed content with metadata indexing and a user-friendly interface."
+licenses:
+  - GPL-3.0
+platforms:
+  - Python
+  - Javascript
+  - Django
+  - Docker
```

```suggestion
platforms:
  - Docker
```

>  Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.

Since the only supported installation method is through Docker/podman https://docs.tubearchivist.com/installation/docker-compose/ (and a few NAS vendor-specific tools)